packages/extension-vscode/src/utils/config.ts

```diff
@@ -0,0 +1,33 @@
+const defaultConfig: import('hint').UserConfig = { extends: ['development'] };
```

Comment:
Can you remind me what are the benefits of using `import('hint')` instead of the tipical: `import { UserConfig } from 'hint'` :)

Reply:
The advantage is that it can only import types, not implementation. Here I want to be absolutely certain we don't accidentally depend on having a copy of `hint` bundled with the extension so I've used the `import('hint')` syntax for all types from `hint` to make that obvious.